src/main/java/com/sap/oss/phosphor/fosstars/model/Artifact.java

```diff
@@ -0,0 +1,26 @@
+package com.sap.oss.phosphor.fosstars.model;
```

Comment:
Let's please move this interface and the child classes to `com.sap.oss.phosphor.fosstars.model.subject.oss`.